js/woo.js

```diff
@@ -93,14 +97,15 @@ module.exports = class woo extends Exchange {
             'urls': {
                 'logo': 'https://user-images.githubusercontent.com/1294454/150730761-1a00e5e0-d28c-480f-9e65-089ce3e6ef3b.jpg',
                 'api': {
+                    'pub': 'https://api-pub.staging.woo.org',
```

Comment:
We cannot use the staging API URL here, it should be a production API URL instead.